diff --git a/Makefile.qemu b/Makefile.qemu
index f18a092..2891c05 100644
--- a/Makefile.qemu
+++ b/Makefile.qemu
@@ -47,6 +47,7 @@ QEMU_CONFIGURE_FLAGS = --prefix=$(S2E_PREFIX)        \
                        --disable-nettle              \
                        --disable-curl                \
                        --disable-opengl              \
+                       --disable-werror              \
                        $(EXTRA_QEMU_FLAGS)
 
 QEMU_DEBUG_FLAGS = --enable-debug
