diff --git a/guest/s2ebios/Makefile b/guest/s2ebios/Makefile
index d2f61ec..9f8f7ae 100644
--- a/guest/s2ebios/Makefile
+++ b/guest/s2ebios/Makefile
@@ -74,7 +74,7 @@ s2e-bios-up.bin : $(BIOS_ASM_SOURCE)
 	$(call quiet-command,$(ASM) -O1 -i$(S2EBIOS)/ -f bin -o $@ $(BIOS_ASM_SOURCE),"  Building $(TARGET_DIR)$@")
 
 s2e-bios-low-c: $(S2E_BIOS_C_OBJ)
-	$(LD) -T $(LD_SCRIPT)  -o $@ $^
+	$(LD) -T $(LD_SCRIPT)  -o $@ $^ -L/usr/lib -L/usr/lib32 -lc
 
 s2e-bios-low-c.bin: s2e-bios-low-c
 	objcopy -O binary --rename-section .data=.rodata,alloc,load,data,contents,readonly $^ $@
